packages/picom/build.sh

```diff
@@ -0,0 +1,12 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/yshui/picom
+TERMUX_PKG_DESCRIPTION="A lightweight compositor for X11"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@Yisus7u7 <jesuspixel5@gmail.com>"
+TERMUX_PKG_VERSION=8.2
+TERMUX_PKG_SRCURL=https://github.com/yshui/picom/archive/refs/tags/vNext.tar.gz
```

Comment:
Please never use *unversioned* tarballs.

Reply:
The latest version doesn't have version ID, so I can't do it

Reply:
Latest is here: https://github.com/yshui/picom/archive/refs/tags/v8.2.tar.gz. The one you picked is a pre-release. Don't use nightlies, pre-releases and so on without good reason.

You can use commits to download tarball without specifying version: https://github.com/yshui/picom/archive/dac85eac10082dfc3df463aaa74b811144e22122.zip.

Source URLs should point to content that doesn't change over time.

Reply:
Solved